usaspending_api/search/delta_models/subaward_search.py

```diff
@@ -0,0 +1,558 @@
+    "sub_ultimate_parent_legal_enti": {"delta": "STRING", "postgres": "TEXT"},
+    "business_type_code": {"delta": "STRING", "postgres": "TEXT"},
+    "business_categories": {"delta": "ARRAY<STRING>", "postgres": "TEXT[]"},
+    "treasury_account_identifiers": {"delta": "ARRAY<INTEGER>", "postgres": "INTEGER[]"},
+    "pulled_from": {"delta": "STRING", "postgres": "TEXT"},
+    "type_of_contract_pricing": {"delta": "STRING", "postgres": "TEXT"},
+    "type_set_aside": {"delta": "STRING", "postgres": "TEXT"},
+    "extent_competed": {"delta": "STRING", "postgres": "TEXT"},
+    "product_or_service_code": {"delta": "STRING", "postgres": "TEXT"},
+    "product_or_service_description": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_legal_entity_country_code": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_legal_entity_country_name": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_legal_entity_county_code": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_legal_entity_county_name": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_legal_entity_zip5": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_legal_entity_city_code": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_legal_entity_congressional": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_perform_scope": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_place_of_perform_country_co": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_place_of_perform_country_name": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_place_of_perform_county_code": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_place_of_perform_county_name": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_place_of_perform_zip5": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_place_of_perform_city_code": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_place_of_perform_congressio": {"delta": "STRING", "postgres": "TEXT"},
+}
+SUBAWARD_SEARCH_DELTA_COLUMNS = {k: v["delta"] for k, v in SUBAWARD_SEARCH_COLUMNS.items()}
+SUBAWARD_SEARCH_POSTGRES_COLUMNS = {k: v["postgres"] for k, v in SUBAWARD_SEARCH_COLUMNS.items()}
+
+SUBAWARD_SEARCH_POSTGRES_VECTORS = {
+    "keyword_ts_vector": ["sub_awardee_or_recipient_legal", "product_or_service_description", "subaward_description"],
+    "award_ts_vector": ["award_piid_fain", "subaward_number"],
+    "recipient_name_ts_vector": ["sub_awardee_or_recipient_legal"],
+}
+SUBAWARD_SEARCH_POSTGRES_COLUMNS.update({col: "TSVECTOR" for col in SUBAWARD_SEARCH_POSTGRES_VECTORS})
+
+subaward_search_create_sql_string = rf"""
+    CREATE OR REPLACE TABLE {{DESTINATION_TABLE}} (
+        {", ".join([f'{key} {val}' for key, val in SUBAWARD_SEARCH_DELTA_COLUMNS.items()])}
+    )
+    USING DELTA
+    LOCATION 's3a://{{SPARK_S3_BUCKET}}/{{DELTA_LAKE_S3_PATH}}/{{DESTINATION_DATABASE}}/{{DESTINATION_TABLE}}'
+"""
+
+subaward_search_load_sql_string = fr"""
+    WITH location_summary AS (
+        SELECT
+            -- DISTINCT ON (UPPER(feature_name), state_alpha)
```

Comment:
Can this comment be removed?

Reply:
Sure thing. Kept it there to inform anyone coming across it what the intent on the partitioning was but it's in the history just in case.